packages/redux/src/contents/utils.ts

```diff
@@ -322,3 +323,21 @@ export const generateSEOPathname = (
 
   return `${query.path}`;
 };
+
+/**
+ * Build a hash with query object received to identify the SEO Files.
+ *
+ * @example
+ * ```
+ * const pathname = generateSEOFilesHash({ name: 'siteSEOFiles', hostID: 1234});
+ * Result of siteSEOFiles!1234;
+ * ```
+ *
+ *
+ * @param query - Object with query parameters applied to search for SEO Files.
+ *
+ * @returns - Hash built to identify the SEO Files for a specific name and hostId.
+ */
+export const generateSEOFilesHash = (query: GetSEOFilesQuery): string => {
```

Comment:
No need to specify the return value here as TS will infer it.

```suggestion
export const generateSEOFilesHash = (query: GetSEOFilesQuery) => {
```

Reply:
Resolved! ✅